outline.c

```diff
@@ -135,25 +135,11 @@ _outline_close(OutlineObject* self, PyObject* args)
     return Py_None;
 }
 
-static PyObject*
-_outline_transform(OutlineObject* self, PyObject* args)
-{
-    double a[6];
-    if (!PyArg_ParseTuple(args, "(dddddd)", a+0, a+1, a+2, a+3, a+4, a+5))
-        return NULL;
-
-    ImagingOutlineTransform(self->outline, a);
```

Comment:
Is it also not used anymore? Should we instead add public API for `outline.transform`?